### PR TITLE
1st module Capstone

### PR DESCRIPTION
**Pull request by Sirvelasque**
- Create 2 pages of the website About and home pages
- Headline seccion with custom background
- Responsive design
- Features section made with javascript
- Custom topic

# link to loom video
- https://www.loom.com/share/dcd10db783484b3395cbb1cac4e03c63

### CODE REVIEW

File: html/about.html [hunk 1]
```diff
@@ -1 +1,137 @@
...
+        </div>
+      </div>
+      <div class="copy_right">
+        <div class="footerItem">
+          <img src="../img/footerlogo.png" alt="footer logo">
+          <h3>Trampoline <br/> Dev Honduras</h3>
+        </div>
+        <div class="footerItem">2022 Trampoline Dev Honduras <br/> some Rights Reserved</div>
+      </div>
+    </footer>
```
Comment (micheaol) [inline html/about.html:134]: Please ensure that the `About Page footer` has a dark background according to the requirement.

File: index.html [hunk 2]
```diff
@@ -3,14 +3,165 @@
...
+        </div>
+      </div>
+    </div>
+
+      <button id="Program-btn">Join TD Global Summit 2022</button>
+      <div class="seeprogram"><a href="#">Se the whole program</a></div>
+    </section>
+    <!--feature spekear-->
+    <section id="features">
+    </section>
```
Comment (micheaol) [inline index.html:143]: Please ensure that the pages look almost identical to the original design.

Your Implementation | Figma Design
:---:|:---:
   
your_design | design

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @Sirvelasque , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️

- _Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Deepakdanger — APPROVED** [review]
Hi @Sirvelasque, 

### Status: Approved ✔️

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**